.github/workflows/docs.yml

```diff
@@ -8,61 +8,231 @@ on:
   schedule:
     - cron:  '0 9 * * 1'
   workflow_dispatch:
-
-jobs:
+    inputs:
+      testing:
+        type: boolean
+        description: Is this a test run that will be built into gamma account?
+        default: false
+        required: false
+      is-new-version:
+        type: boolean
+        description: Are you deploying new version? If so, enter the new version number (ex. 28.0). If not, skip the rest of the parameters. 
+        default: false
+        required: true
+      version-number:
+        type: string 
+        description: What is the new version number for the website?
+        default: dev
+        required: false
+      djl-tag: 
+        type: string
+        description: The commit SHA (e.g. 6b642e1) for the tag of the version of djl.
+        default: 'DJL SHA'
+        required: false
+      djl-serving-tag:
+        type: string
+        description: The commit SHA (e.g. 6b642e1) for the version of djl-serving.
+        default: 'DJL Serving SHA'
+        required: false
+      djl-demo-tag:
+        description: The commit SHA (e.g. 6b642e1) for the version of djl-demo. 
+        default: 'DJL Demo SHA'
+        required: false
+      
+permissions:
+  id-token: write 
+  contents: read 
+  
+jobs:    
   documentation:
-    if: github.repository == 'deepjavalibrary/djl'
+    # Change line below 
+    if: github.repository == 'Varun-Dutta/djl'  
     runs-on: ubuntu-latest
     steps:
+        
+    #Phase I: Set-up
+    #This phase sets up all of the necessary dependencies. This includes authenticating the workflow, installing Java and Python, and adding the tools used to build the documentation website.  
+      - name: Configure AWS Test Credentials
```

Comment:
This should be in later steps to avoid credential accidentally leaked to build process

Reply:
Implemented this change.